store/redis_supplier.go

```diff
@@ -132,3 +132,23 @@ func (s *RedisSupplier) ReactionPermanentDeleteBatch(ctx context.Context, endTim
 	// Ignoring this. It's probably OK to have the emoji slowly expire from Redis.
 	return s.Next().ReactionPermanentDeleteBatch(ctx, endTime, limit, hints...)
 }
+
+func (s *RedisSupplier) RoleSave(ctx context.Context, role *model.Role, hints ...LayeredStoreHint) *LayeredStoreSupplierResult {
+	// TODO: Redis Caching.
```

Comment:
Are these `TODO`s slated for a later iteration or were they forgotten? Just want to be sure.

Reply:
I'm intending to implement them later as XYZ-59.